docs/_docs/hivemq/policies.md

```diff
@@ -0,0 +1,287 @@
+---
+nav_order: 1
+redirect_from: /docs/hivemq/policies.html
+---
+
+# Policies
+
+***
+
+The policies command of the HiveMQ command line offers a set of commands to work with policies using a HiveMQ Data
+Validation API endpoint.
+
+```
+mqtt hivemq policies
+```
+
+***
+
+## Commands
+
+| Command | Explanation                         |
+|---------|-------------------------------------|
+| create  | See [Create Policy](#create-policy) |
+| get     | See [Get Policy](#get-policy)       |
+| list    | See [List Policies](#list-policies) |
+| delete  | See [Delete Policy](#delete-policy) |
+
+***
+
+## Options
+
+### Help Options
+
+{% include options/help-options.md defaultHelp=true %}
+
+***
+
+# Create Policy
+
+*** 
+
+Create a new policy and upload it to a HiveMQ node.
+The policy definition may be provided either directly from the command line or from a file.
+
+```
+mqtt hivemq policies create
+```
+
+***
+
+## Simple Example
+
+```
+mqtt hivemq policies create --file my-policy.json 
```

Comment:
Is there a reason why you cant provide an `-i` / `--id` here?

Reply:
Policy ids are specified within the policy JSON.